app/models/game.rb

```diff
@@ -99,15 +99,15 @@ def defender_response!
     end
   end
 
-  def confirm_user(user)
+  def confirm_user(current_user)
     raise "Cannot confirm if game is incomplete" if incomplete? || challenged?
     tournament.with_lock do
       lock!
 
       total = 0
       confirmed = 0
       game_ranks.each do |game_rank|
-        game_rank.confirm if game_rank.user == user
+        game_rank.confirm if game_rank.user == current_user || tournament.owner == current_user
```

Comment:
Games are setup to support more than 2 players. This would mean a tournament owner could never confirm their own game rank without confirming the whole thing.

Reply:
Ah was only looking at this from a 2 player game